### ISSUE TITLE
Custom profile fields

### ISSUE THREAD

**Issue by renfordt** [TangoBB/TangoBB#50]
Every forum needs different profile fields, so why should the admin not create easily their own profile fields. 
The admin can define:
- title
- fieldsize (simple input, textbox etc)
- data type

After it users will see those fields in their profile.

**Comment by KustaJuspin** [comment]
Maybe,this forum sofeware will was forgotten gradually......

**Comment by N8boy** [comment]
Yes this forum seems to be dead. Take a look at http://ikobb.com or Ikobb.de it's a software made orinally out of TangoBB but we have reworked it completely
